css/main.css

```diff
@@ -1,11 +1,6 @@
-body {
-	padding-top: 3em;
-	max-width: 45em;
```

Comment:
Sorry if I'm not clear. The discussion is not about the amount of padding, it is about where does the instruction for padding fits  ;)

I'm thinking `max-width`, `line-height`, `font-size` are all part of making the text-based interface usable.

`padding`, however, is dependent on the host webpage content. I'm not sure we can make assumptions about what's the best amount of padding.

So, for our `index.html`, let's put what's the best for readability. We had `3em`, if you prefer `1.5em`, it is fine too.
But I don't think we should have `3em` for the distributed widget; and I'm considering not setting a `padding` at all, and leaving spacing from the outside content to the host altogether.

I understand your argument about the “necessary title of introduction text”, but I don't feel like we can be sure all our clients will have such a title. What if they include the widget as a popover, for example?